jsonb-generator/src/main/java/io/avaje/jsonb/generator/FieldProperty.java

```diff
@@ -278,6 +278,9 @@ void writeFromJsonSetter(Append writer, String varName, String prefix, String nu
       writeFromJsonUnmapped(writer, varName);
       return;
     }
+    if (constructorParam) {
```

Comment:
This looks like a bug fix?

Reply:
When I added the test class it failed because this wasn't there